core/modules/email/email.install

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
```

Comment:
Minor, let's remove the extra returns from the top of all files. A long time ago Drupal included CVS version strings at the top of files, but that hasn't been the case in a long time. The top of files should be:
```
<?php
/**
 * @file
 * Description of file.
 */
```